### PR TITLE
issue template: remove browser and version questions

### PR DESCRIPTION
**Pull request by gucio321**
AbyssEngine isn't related to browser at all so asking for browser/version doesn't make sens here.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -24,9 +24,7 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
```
Comment (essial) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: Don't we still need version?

Comment (gucio321) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: I think i wanted to keep os point...

Comment (gucio321) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: ok, now it is ok imo

**Comment by gucio321** [comment]
sorry, I forgot about this PR
@essial it should be ready now

**Pull request closed without merging**